src/pages/careers/_model/_locations/_locations.ts

```diff
@@ -344,6 +347,7 @@ export const minsk: LocationsType = {
         'Girl having a chat ',
     ],
     map: 'ChIJF9lVNV_P20YRukiwrfpLP4E',
+    badge: 'minsk_great_place_to_work_badge',
```

Comment:
@konstantinosG-deriv Can we add `badge_alt` for these new badges as well 🙏

Reply:
Oh yeah will do